src/deployers/PluginsDeployerFactory.cpp

```diff
@@ -41,6 +43,10 @@ std::vector<std::shared_ptr<PluginsDeployer>> PluginsDeployerFactory::getDeploye
     }
 
     if (moduleName == "network") {
+        if (qtMajorVersion == 6) {
```

Comment:
I'd rather not instantiate the basic deployer manually, but would put the check in the top `if` clause. That way, it'll fall back to this one as well.

```cpp
    if (moduleName == "network" && qtMajorVersion < 6) {
```

Reply:
Applied